.github/workflows/docs.yml

```
@@ -0,0 +1,33 @@
+name: Build and deploy JavaDocs
+
+on:
+  push:
+    branches:
+      - dev
```

Docs should only change when releasing a new version
```suggestion
      - master
```

---

Changed 👍